systemtests/utils/utils.go

```diff
@@ -154,6 +153,8 @@ func StartNetPlugin(t *testing.T, nodes []TestbedNode, nativeInteg bool) {
 
 // StartNetmasterWithFlags starts netplugin on specified testbed nodes with specified flags
 func StartNetmasterWithFlags(t *testing.T, node TestbedNode, flags map[string]string) {
+	time.Sleep(5 * time.Second)
```

Comment:
So one of the changes I made here was that before, at least in the etcd case, all members of the cluster were actually on host #1 -- you should be able to verify this with `set -x` on in the vagrantfile provisioning script before etcd spawns.

I've watched the logs with this branch at this latest rev, this does not happen anymore, at least I can see. I'm guessing that what you might be seeing is normal periodic leader election that would not occur before.